frontend/package.json

```diff
@@ -9,6 +9,7 @@
     "axios": "0.23.0",
     "bootstrap": "4.6.0",
     "classnames": "2.3.1",
+    "convert": "4.4.2",
```

Comment:
extra dependency to convert between various units of measure - e.g. meters -> acres etc